app/store/configureStore.dev.js

```diff
@@ -1,23 +1,18 @@
-import { createStore, applyMiddleware, compose } from 'redux';
```

Comment:
Is there really any benefit in having separate files for configuring the store in dev vs prod, especially now that RTK is handling setting up the DevTools Extension?

Reply:
I'm not sure why the original decision was made to separate it into separate files.  I have updated this PR to include a consolidation to a single store.js setup file.  It does still include a couple process.env.NODE_ENV checks though which is what they might have originally been trying to avoid?